taliforth.asm

```diff
@@ -111,16 +142,61 @@ dodefer:
 
                 jmp (tmp2)      ; This is actually a jump to the new target
 
-defer_error:
-                ; """Error routine for undefined DEFER: Complain and abort"""
-                lda #err_defer
-                jmp error
+
+domarker:
+                ; keep actual code next to MARKER since save/restore mirror each other
+                jmp marker_runtime
```

Comment:
this adds some cycles to marker but on balance it seemed to make sense to keep the read/write marker code near each other.